smart_contracts/fabric/custompin-sample/batchpin/batchpin.go

```diff
@@ -0,0 +1,63 @@
+package batchpin
```

Comment:
I ended up copying this file between the FireFly chaincode project and this one. I wanted to have a single shared copy of the file somewhere in this repo, but couldn't figure out a way to make that work with the Fabric chaincode packaging.